cobalt/cobalt_paths.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CONTENT_SHELL_BROWSER_SHELL_PATHS_H_
+#define CONTENT_SHELL_BROWSER_SHELL_PATHS_H_
+
+#include "build/build_config.h"
```

Comment:
Why is this include needed here?

Reply:
This was copied from content/shell/browser/shelL_paths.h, but it doesn't need it. removed.